spotify_player/src/state/ui/popup.rs

```diff
@@ -1,18 +1,35 @@
 use crate::{command, state::model::*};
 use tui::widgets::ListState;
 
+#[derive(Debug)]
+pub enum PlaylistCreateCurrentField {
+    Name,
+    Desc,
+}
+
 #[derive(Debug)]
 pub enum PopupState {
-    CommandHelp { scroll_offset: usize },
-    Search { query: String },
+    CommandHelp {
+        scroll_offset: usize,
+    },
+    Search {
+        query: String,
+    },
     UserPlaylistList(PlaylistPopupAction, ListState),
     UserFollowedArtistList(ListState),
     UserSavedAlbumList(ListState),
     DeviceList(ListState),
     ArtistList(ArtistPopupAction, Vec<Artist>, ListState),
     ThemeList(Vec<crate::config::Theme>, ListState),
     ActionList(ActionListItem, ListState),
-    Queue { scroll_offset: usize },
+    Queue {
+        scroll_offset: usize,
+    },
+    PlaylistCreate {
```

Comment:
It's better to have a focus state for `PlaylistCreate` so that user can know which component is in focus. See the **search page** implementation and its UI for reference.